dashboard/app/graphs.go

```diff
@@ -241,15 +241,24 @@ func handleHeatmap(c context.Context, w http.ResponseWriter, r *http.Request, f
 	})
 }
 
-func githubTorvaldsLinuxURI(filePath string, rc covermerger.RepoCommit) string {
-	return fmt.Sprintf("https://raw.githubusercontent.com/torvalds/linux/%s/%s", rc.Commit, filePath)
+func makeProxyURIProvider(url string) covermerger.FuncProxyURI {
+	if url == "" {
```

Comment:
Maybe just disable the feature if the url is not provided?

Reply:
Changed to 404 in case of empty config param.